docs/tutorials/02-channel-upgrades/05-upgrade-channel.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Upgrade channel
+sidebar_label: Upgrade channel
+sidebar_position: 5
+slug: /channel-upgrades/upgrade-channel
+---
+
+# Upgrade the ICS 20 transfer channel
+
+## Start the relayer
+
+We start the relayer:
+
+```bash
+hermes --config config.toml start
+```
+
+## Initiate the upgrade
+
+The [initiation of the upgrade process is authority-gated](https://ibc.cosmos.network/main/ibc/channel-upgrades#governance-gating-on-chanupgradeinit), so in this example the message to initiate the upgrade will execute when a governance proposal passes. The contents of the governance proposal are:
+
+```json title=proposal.json
```

Comment:
This file was not included in Folder Structure

Reply:
Good one, I will add it.